Libplanet.Tools/Program.cs

```diff
@@ -5,6 +5,7 @@ namespace Libplanet.Tools
 {
     [HasSubCommands(typeof(Apv), Description = "App protocol version utilities.")]
     [HasSubCommands(typeof(Key), Description = "Manage private keys.")]
+    [HasSubCommands(typeof(Mpt), Description = "Merkle Trie utilities.")]
```

Comment:
It's a trivia but: even though the sub-command `mpt` has a P in its name, the description doesn't mention Patricia.